eng/pipelines/linux.yml

```
@@ -50,7 +51,7 @@ jobs:
              _buildScriptPrefix: ''
              _buildExtraArguments: /p:RuntimeOS=linux-musl
 
-          ${{ if eq(parameters.isOuterloopBuild, 'true') }}:
+          ${{ if or(eq(parameters.testfilter, 'outerloop'), eq(parameters.testfilter, 'all')) }}:
```

this is wrong because the parameter is defined with camel case and here is all lowercase.

```suggestion
          ${{ if or(eq(parameters.testFilter, 'outerloop'), eq(parameters.testFilter, 'all')) }}:
```